websites/L/Lidraughts/presence.ts

```diff
@@ -0,0 +1,79 @@
+const presence = new Presence({
+		clientId: "1046315725960265839",
+	}),
+	pages: {
+		[name: string]: string;
+	} = {
+		"/": "Home",
+		"/tournament": "Tournaments",
+		"/simul": "Simultaneous exhibitions",
+		"/training/coordinate": "Coordinate training",
+		"/study": "Study",
+		"/variant": "Rules & Variants",
+		"/tv": "Lidraughts TV",
+		"/games": "Current games",
+		"/streamer": "Streamers",
+		"/broadcast": "Broadcasts",
+		"/player": "Players",
+		"/team/all": "Teams",
+		"/forum": "Forums",
+		"/faq": "FAQ",
+		"/analysis": "Analysis board",
+		"/editor": "Board editor",
+		"/paste": "Import game",
+		"/games/search": "Advanced search",
+	};
```

Comment:
Use a Record as shown in the example instead
https://github.com/PreMiD/Presences/blob/57fc80415207783fd9fbe6fcaf421983180fe64b/websites/C/Caards/presence.ts#L16-L34

Reply:
Hey, thanks for the suggestions, sorry for the late response, but I'll implement them when I have the time 🙂